spring-integration-core/src/main/java/org/springframework/integration/context/IntegrationContextUtils.java

```diff
@@ -77,6 +77,8 @@
 	public static final String TO_STRING_FRIENDLY_JSON_NODE_TO_STRING_CONVERTER_BEAN_NAME =
 			"toStringFriendlyJsonNodeToStringConverter";
 
+	public static final String INTEGRATION_LIFECYCLE_ROLE_CONTROLLER = "lifecycleRoleController";
```

Comment:
Wouldn't `integrationLifecycleRoleController` be better name?
At least for those cases when some target application has done something with similar name...